src/execution_plan/ops/op_merge.h

```diff
@@ -16,20 +16,22 @@
  * If the path is not found, it will be created, making new instances of every path variable
  * not bound in an earlier clause in the query. */
 typedef struct {
-	OpBase op;                               // Base op.
-	OpBase *match_stream;                    // Child stream that attempts to resolve the pattern.
-	OpBase *create_stream;                   // Child stream that will create the pattern if not found.
-	OpBase *bound_variable_stream;           // Optional child stream to resolve previously bound variables.
-	Argument *match_argument_tap;            // Argument tap to populate Match stream with bound variables.
-	Argument *create_argument_tap;           // Argument tap to populate Create stream with bound variables.
-	Record *input_records;                   // Records generated by the bound variable stream.
-	Record *output_records;                  // Records to be emitted by this operation.
-	rax *on_match;                           // Updates to be performed on a successful match.
-	rax *on_create;                          // Updates to be performed on creation.
-	raxIterator on_match_it;                 // Iterator for traversing ON MATCH update contexts.
-	raxIterator on_create_it;                // Iterator for traversing ON CREATE update contexts.
-	dict *node_pending_updates;              // Pending updates to apply, generated 
-	dict *edge_pending_updates;              // Pending updates to apply, generated 
+	OpBase op;                               // Base op
+	OpBase *match_stream;                    // Child stream that attempts to resolve the pattern
+	OpBase *create_stream;                   // Child stream that will create the pattern if not found
+	OpBase *bound_variable_stream;           // Optional child stream to resolve previously bound variables
+	Argument *match_argument_tap;            // Argument tap to populate Match stream with bound variables
+	Argument *create_argument_tap;           // Argument tap to populate Create stream with bound variables
+	Record *input_records;                   // Records generated by the bound variable stream
+	Record *output_records;                  // Records to be emitted by this operation
+	rax *on_match;                           // Updates to be performed on a successful match
+	rax *on_create;                          // Updates to be performed on creation
+	raxIterator on_match_it;                 // Iterator for traversing ON MATCH update contexts
+	raxIterator on_create_it;                // Iterator for traversing ON CREATE update contexts
+	dict *node_pending_updates;              // Pending updates to apply, generated
+	dict *edge_pending_updates;              // Pending updates to apply, generated
+	RG_Matrix add_labels;                    // Labels to add to nodes
+	RG_Matrix remove_labels;                 // Labels to remove from nodes
```

Comment:
if you've already modified this, please make sure the spacing is correct, 2 char from the longest line.